example/spec/parallel/ingestGranule/IngestUMMGSuccessSpec.js

```diff
@@ -299,7 +374,16 @@ describe('The S3 Ingest Granules workflow configured to ingest UMM-G', () => {
     it('updates the UMM-G JSON file in S3 with new paths', async () => {
       const updatedUmm = await getUmmObject(newS3UMMJsonFileLocation);
 
-      const relatedUrlDifferences = differenceWith(updatedUmm.RelatedUrls, originalUmm.RelatedUrls, isEqual);
+      const relatedUrlDifferences = updatedUmm.RelatedUrls.filter((urlObject) => {
```

Comment:
This had to be done because `updatedUmm` is generated by a move operation on the granules API. This operation uses the domain configured for the stack (`example.com` by default) for the URLs and there is no way I can see to change that, since it runs on the deployed API for the stack. `originalUmm` from the workflow execution will have distribiution API URLs starting with `http://localhost:5002`. So to compare the URLs for differences I have to isolate the distribution API URLs from `updatedUMM` and change their host to `localhost:5002`

Reply:
👍